sbt-scalajs-bundler/src/main/scala/scalajsbundler/ExternalCommand.scala

```diff
@@ -33,20 +34,47 @@ object Npm extends ExternalCommand("npm")
 object Yarn extends ExternalCommand("yarn")
 
 object ExternalCommand {
+  private val yarnOptions = List("--non-interactive", "--mutex", "network")
+  private def syncYarnLockfile(baseDir: File, installDir: File, logger: Logger)(yarnCommand: => Unit):Unit = {
+    val sourceLockFile = baseDir / "yarn.lock"
+    val targetLockFile = installDir / "yarn.lock"
+    if(sourceLockFile.exists()) {
+      logger.info("Using lockfile " + sourceLockFile)
+      IO.copyFile(sourceLockFile, targetLockFile)
+    }
+
+    yarnCommand
+
+    if(targetLockFile.exists()) {
+      logger.info("Wrote lockfile to " + sourceLockFile)
```

Comment:
Maybe put that line to the `debug` level?

Reply:
good idea, I'll change it.